mne/beamformer/_lcmv.py

```diff
@@ -656,12 +676,15 @@ def lcmv(evoked, forward, noise_cov=None, data_cov=None, reg=0.05, label=None,
     .. [2] Sekihara & Nagarajan. Adaptive spatial filters for electromagnetic
            brain imaging (2008) Springer Science & Business Media
     """
+    info = evoked.info
+    _deprecate_picks(info, picks)
```

Comment:
you do not want to modify `evoked.info`, but this function operates inplace so it will. You can do `info = evoked.info.copy()` above if you want, or better, do `info = _deprecate_picks(info, picks)` and only copy if necessary